x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/waterfall_helpers/waterfall_helpers.test.ts

```diff
@@ -717,4 +718,46 @@ describe('waterfall_helpers', () => {
       expect(getClockSkew(child, parent)).toBe(0);
     });
   });
+
+  describe('getHasOrphanTraceItems', () => {
+    const myTransactionItem = {
+      processor: { event: 'transaction' },
+      trace: { id: 'myTrace' },
+      transaction: {
+        id: 'myTransactionId1',
+      },
+    } as WaterfallTransaction;
+
+    it('should return false if there is not orphan items', () => {
```

Comment:
```suggestion
    it('should return false if there are no orphan items', () => {
```

Reply:
Forgot that after I changed item with items, fixed 👍